mh/constants.py

```diff
@@ -21,19 +21,24 @@
 
 import struct
 from other.utils import pad
+from mh.time_utils import current_tick, TICKS_PER_CYCLE, get_fog_start,\
+    get_sandstorm_start, get_sandstorm_end
 from quest_utils import QUEST_EVENT_JUMP_FOUR_JAGGI, QUEST_EVENT_BLOOD_SPORT,\
     QUEST_EVENT_MERCY_MISSION, QUEST_EVENT_THE_PHANTOM_URAGAAN, QUEST_EVENT_WORLD_EATER,\
     QUEST_EVENT_WHERE_GODS_FEAR_TO_TREAD, QUEST_EVENT_GREEN_EGGS
 
 
-def make_binary_type_time_events(state=0):
-    # TODO: Use the server in-game time and handle this time array properly
-    if state == 1:
-        return struct.pack(">iii", 0, -1, 0)  # Fog
-    elif state == 2:
-        return struct.pack(">iii", 0, 0, -1)  # Kujira
-    else:
-        return struct.pack(">iii", 0, 0, 0)
+def make_binary_type_time_events():
+    """
+    If the first int is less than the gametime at server login:
+      If the second int is greater than the gametime at server login, fog
+      Otherwise, if the third int is greater than the gametime at server login, sandstorm
+    FIRST INT: Start of fog (epoch seconds)
+    SECOND INT: Start of Jhen event (epoch seconds)
+    THIRD INT: End of Jhen event (epoch seconds)
+    """
```

Comment:
This docstring should be moved to the util function in `mh.time_utils`, IMHO. So it would look like something like this where `get_time_events` is the function returning the tuple of events:
```python
def make_binary_type_time_events():
   return struct.pack(">III", *get_time_events())
```

Reply:
Addressed this while addressing a previous comment. Also moved the docstring.